README.md

```diff
@@ -337,6 +337,18 @@ Let's see how it works by following a simple example:
 Imagine that we have a `updatePet` operation defined in the `petstore.json` specification file and secured with the `petstore_auth` security scheme.
 The code below shows a simple example of the usage of this operation in a user-programmed service.
 
+## Headers propagation
```

Comment:
This is not the right place for this section. You placed the "Headers propagation" section in the middle of the "Authorization Token Propagation" section. I think you can put it before the "## Circuit Breaker" section.

Reply:
ok, will fix